### PR TITLE
feat: show message data for ERC20 function calls

### PR DESCRIPTION
**Pull request by troggy**
Please remember the [Contributing Guidelines](https://github.com/leapdao/meta/blob/master/CONTRIBUTION.md) :heart:

## What does this PR do?

image

### CODE REVIEW

File: src/routes/safe/components/Transactions/TxsTable/ExpandedTx/TxDescription/index.tsx [hunk 1]
```diff
@@ -209,6 +214,25 @@ const CustomDescription = ({ amount = 0, classes, data, recipient }: any) => {
...
+          <Bold>Decoded data:</Bold>
+          <br />
+          {functionName}
+          {'('}
+          <br />
+          {functionArgs.map((value, i) => (
+            <>
+              <span style={{ marginLeft: '14px' }}>
+                {value}
+                {i < functionArgs.length - 1 && <>{','}</>}
```
Comment (roleengineer) [inline src/routes/safe/components/Transactions/TxsTable/ExpandedTx/TxDescription/index.tsx]:
```suggestion
                {i < functionArgs.length && <>{','}</>}
```

Comment (troggy) [reply src/routes/safe/components/Transactions/TxsTable/ExpandedTx/TxDescription/index.tsx]: 🤦‍♂️
thanks!

**Comment by troggy** [comment]
image

**Review by johba37 — APPROVED** [review]
great work guys 👏

**Comment by johba37** [comment]
big numbers are cut off an small screens.
image

**Comment by troggy** [comment]
image

**Pull request merged**